### ISSUE TITLE
Converted Group Loses Inputs

### ISSUE THREAD

**Issue by wyrde** [pythongosssss/ComfyUI#4]
A group converted to node loses most of its inputs.

![2023-04-28 10_21_47-ComfyUI — Mozilla Firefox](https://user-images.githubusercontent.com/9657443/235174173-c434a76e-44f0-4489-b8bc-3c4c58783fce.png)
![2023-04-28 10_22_02-ComfyUI — Mozilla Firefox](https://user-images.githubusercontent.com/9657443/235174186-c9fde3a7-809c-41d6-8a62-e3a171c342b1.png)

Does not allow reconnecting the inputs from the previous routes.

Replaced the ksampler in case it was the seed thing.

Removed the INT bypass in case it didn't like the unconnected route.

Not sure what is happening. The first group node works fine.

workflow:
[groupnodetest06.zip](https://github.com/pythongosssss/ComfyUI/files/11354453/groupnodetest06.zip)

**Comment by pythongosssss** [comment]
![image](https://user-images.githubusercontent.com/125205205/235205029-113adbd4-bdfb-4e90-b1d9-a44ba80849ad.png)
When i load up that graph, none of those inputs are actually connected

There is however an issue with reroutes connected to other reroutes so looking into that.

**Comment by pythongosssss** [comment]
I've made a few reroute fixes, if you could please give it a check when you get a chance, thanks!

As mentioned, I couldnt reproduce your exact issue - but did find a bunch of similar ones so its hopefully fixed

**Comment by wyrde** [comment]
Gave it a few shots and was not able to break it \o/

During the process, I think I figured out what happened: there was a sneaky reroute hidden behind a node.  You fixed that--they make more inputs now.

Now I'm wondering how many issues were caused by that stupid hidden reroute. Sheesh. Well, not an issue now. (:

![2023-04-29 09_53_33-ComfyUI — Mozilla Firefox](https://user-images.githubusercontent.com/9657443/235306534-937886b6-164f-4728-8a28-2b037526051e.png)
![2023-04-29 09_53_58-ComfyUI — Mozilla Firefox](https://user-images.githubusercontent.com/9657443/235306540-fd4b79b3-ece6-4685-a763-638265c8a023.png)

yay!
